hloc/triangulation.py

```diff
@@ -126,54 +111,62 @@ def geometric_verification(
     logger.info("Performing geometric verification of the matches...")
 
     pairs = parse_retrieval(pairs_path)
-    db = COLMAPDatabase.connect(database_path)
```

Comment:
I don't like that this introduces such a large diff because of the indentation change, could you instead modify the function definition to pass a database object and call open_colmap_database in the parent scope?

Reply:
Thank you!